src/NzbDrone.Common/Cache/Cached.cs

```diff
@@ -100,6 +100,14 @@ public void Clear()
             _store.Clear();
         }
 
+        public void ClearExpired()
```

Comment:
This was added to purse recently,
https://github.com/kayone/Purse/commit/a12f51bea15ff4f3ca3a9373e28db5cfd32f3f3d